### ISSUE TITLE
@import from a node module

### ISSUE THREAD

**Issue by netopolit** [kentaroi/eleventy-sass#6]
Is there a way to @import from a node module?
Thank you

**Comment by kentaroi** [comment]
Yes, there is.

It's quite easy. Just use [loadPaths](https://github.com/kentaroi/eleventy-sass#loadpaths-and-includes-keys) in your settings.


For example, the way to use the Bootstrap library is as follows:
```JavaScript
// .eleventy.js
module.exports = function(eleventyConfig) {
  eleventyConfig.addPlugin(sass, {
    sass: {
      loadPaths: ["./node_modules/bootstrap/scss", "./src/_includes"]
    }
  });
};
```

```scss
// your Sass/SCSS file
@import "bootstrap";
```

See https://github.com/kentaroi/eleventy-sass-bootstrap-sample for more details.

**Comment by netopolit** [comment]
@kentaroi Thank you very much.

**Comment by kentaroi** [comment]
@netopolit It's my pleasure.

**Comment by netopolit** [comment]
I just got a chance to try it and realized I already tried it. I am trying to import [sass-mq](https://github.com/sass-mq/sass-mq) mixin but but I just can't do it with this method. I'm probably missing something trivial. The directory structure of node module looks like this, if it helps:

<img width="220" alt="CleanShot 2022-05-14 at 23 44 14@2x" src="https://user-images.githubusercontent.com/131025/168449123-5b23c4e6-a5e5-493e-9404-93416759302e.png">
